tasks/metalsmith.js

```diff
@@ -142,6 +142,7 @@ function build(buildCount){
 
 
         // START THE BUILD!
+        // TODO: what's this variable name doing?
```

Comment:
Which variable?

Reply:
Ah, `colophonemes`... A hangover from a veeery long time ago and lots of copypasta 🍝

More specifically, this should be called `metalsmith`. It's possible to run a single chain of calls without the assignment by using the constructor:

```